src/Test.UnitTests.Sarif/Core/StackTests.cs

```diff
@@ -38,8 +38,9 @@ public void Stack_CreateFromException()
             {
                 File.Create(Path.GetInvalidFileNameChars()[0].ToString(), 0);
             }
-            catch (IOException exception)
+            catch (Exception exception)
```

Comment:
Exception to our Exception handling guidelines ? ([https://dev.azure.com/mseng/1ES/_wiki/wikis/Security%20Tools%20Detections%20Team/34629/Team-coding-standards-guidelines](https://dev.azure.com/mseng/1ES/_wiki/wikis/Security%20Tools%20Detections%20Team/34629/Team-coding-standards-guidelines)) 

There is one article about catch multiple exceptions:
[https://stackoverflow.com/questions/136035/catch-multiple-exceptions-at-once](https://stackoverflow.com/questions/136035/catch-multiple-exceptions-at-once)

Reply:
Well, this is a weird one. This test is designed to raise an exception and it doesn't really matter which one. It also isn't a concern that we swallow all exceptions because we're not in product code, we're testing the literal exceptional path.

So it is the 'exception that proves the rule.' Haha. Still, please note that you review feedback to add an exception filter that's constrained to the IOException and ArgumentException *does* meet our strict exception standard. A different exception type will fail the test... So thanks for the note!